docs/maui/essentials/file-saver.md

```diff
@@ -7,20 +7,22 @@ ms.date: 12/11/2022
 
 # FileSaver
 
-The `FileSaver` provides the ability to select target folder and save files to the file system.
+The `FileSaver` provides the ability to select a target folder and save files to the file system.
 
 ![Screenshot of an FileSaver on macOS](../images/essentials/file-saver-mac.png "FileSaver on macOS")
 
-The following preconditions required for the `FileSaver`:
+The following preconditions are required for the `FileSaver`:
 # [Android](#tab/android)
 
-Add permissions to `AndroidManifest.xml`:
+If your target device API level is less than 33 add permissions to `AndroidManifest.xml`:
```

Comment:
What if my target is higher? Then I don't need to add permissions? Do I need different permissions?

Reply:
in our sample app, we have only these 2 permissions and the app works on Android 33. I added a link to Android permissions docs, if something changes for the new APIs, we don't need to change our docs